packages/foundry/src/create.ts

```diff
@@ -0,0 +1,77 @@
+import { basename } from "node:path";
+
+import { cloneProjectTemplate } from "./cloneProjectTemplate.js";
+import { replaceTokens } from "./replaceTokens.js";
+
+const BaseRepositoryAddress = "workleap/wl-foundry-cli/templates";
+
+type TemplateId = "host-application" | "remote-module" | "static-module";
+
+interface TemplateDetails {
+    repositoryUrl: string;
+    action: (outputDirectory: string, options: Record<string, string>) => Promise<void>;
+}
+
+function formatScope(scope: string) {
+    let formattedScope = scope;
+
+    if (scope) {
+        if (formattedScope[0] !== "@") {
```

Comment:
I don't think the generator should fix input issue.

The prompt should ask for a scope starting with "@".

Also, I don't think the generator should be coded defensively, it should assume that the prompt inputs are correct. Otherwise everything is done twice which doesn't make much sense in our case.